clouddriver-api/src/main/java/com/netflix/spinnaker/clouddriver/security/AccountDefinitionRepository.java

```diff
@@ -62,18 +62,26 @@ List<? extends CredentialsDefinition> listByType(
 
   /**
    * Creates a new account definition using the provided data. Secrets should use {@code
-   * SecretEngine} encrypted URIs (e.g., {@code
-   * encrypted:secrets-manager!r:us-west-2!s:my-account-credentials}) when the underlying storage
+   * UserSecretReference} encrypted URIs (e.g., {@code
+   * secret://secrets-manager?r=us-west-2&s=my-account-credentials}) when the underlying storage
```

Comment:
At least for now, describe both `encrypted:` and `secret://` and perhaps describe a migration path?

Reply:
...and an end-of-life plan for `encrypted:` if possible.

Reply:
Re-added support for this. Still need an EOL plan, though not immediately urgent. Or we could keep this as an optional feature since it sort of matches the existing behavior with Spring Cloud Config encrypted properties.

Reply:
Thanks.  Does it make sense to include both `encrypted:` and `secret://` in the javadoc?

Reply:
Done in https://github.com/spinnaker/clouddriver/pull/5719/files/9879aadf0a33a02e0742c266f3256fc1d89d1ecd..27eb955f9c90fe46de2ffa427b54a07b926f0e90